docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   minio:
-    image: minio/minio:RELEASE.2023-03-13T19-46-17Z
+    image: quay.io/minio/minio:RELEASE.2023-08-31T15-31-16Z
```

Comment:
FYI - I had to update this since there was a bugfix on the API, where `region` was ignored when creating remote target

Reply:
All good 👍 

Would there be any impact of setting these lines to `image: minio/minio:RELEASE.2023-08-31T15-31-16Z`? Was quay.io a requirement?

Reply:
ℹ️ Updated the image paths. Everything looks good!